modules/update-service-create-service-web-console.adoc

```diff
@@ -0,0 +1,34 @@
+[id="update-service-create-service-web-console_{context}"]
+= Creating an OpenShift Update Service application by using the web console
+
+You can use the {product-title} web console to create an Update Service application by using the OpenShift Update Service Operator.
+
+.Prerequisites
+
+* The OpenShift Update Service Operator has already been installed.
+
+.Procedure
+
+To create an OpenShift Update Service application by using the {product-title} web console:
+
+. In the {product-title} web console, click *Operators* -> *Installed Operators*.
+
+. Choose *OpenShift Update Service* from the list of available Operators.
+
+. Click the *Update Service* tab.
+
+. Click *Create UpdateService*.
+
+. Enter a name in the *Name* field, for example, `service`.
+
+. Enter the local pullspec in the *Graph Data Image* field to the graph-data container image created in xref:modules/update-service-graph-data.adoc#update-service-graph-data_{context}[Creating the OpenShift Update Service graph data container image], for example, `registry.example.com/openshift/graph-data:latest`.
+
+. Enter the local registry and repository in the *Releases* field, for example, `registry.example.com/openshift/release`.
+
+. Enter `2` in the *Replicas* field.
+
+. Click *Create* to create the OpenShift Update Service application.
+
+. Verify the OpenShift Update Service application:
++
+** Ensure that the *Update Service* application is listed with *Status Condition: ReconcileCompleted*.
```

Comment:
If the registry accessed to OSUS requires additional CA, then the CA needs to be set up by following [Setting up additional trusted certificate authorities for builds](https://docs.openshift.com/container-platform/4.6/builds/setting-up-trusted-ca.html). In this case, the Update Service application is listed with Status `Conditions: ReconcileCompleted, RegistryCACertFound`. I feel it deserves a note. WDYT?
![Uploading Screenshot from 2021-03-25 15-04-24.png…]()

Reply:
Agree. Modified check.